### ISSUE TITLE
satnogs-extras/strf_cronjobs.py update

### ISSUE THREAD

**Issue by satcolintel5** [darksidelemm/satnogs-extras#5]
Hello,

Firstly, thanks for trail blazing all of this! Not sure if this is something you are still maintaining but I figured it's worth a shot.

[satnogs-extras](https://github.com/darksidelemm/satnogs-extras)

strf_cronjobs.py is generating invalid times  -- times for observations that have already completed , sometimes days ago --  due to 'vetted_status=unknown' in line 42. -- probably due to a change in the API since this code was written.

https://github.com/darksidelemm/satnogs-extras/blob/3f3d9482365aacc4083e36eb9e6e6ba77e3d0c9b/scripts/strf_cronjobs.py#L42

replacing 'vetted_status=unknown' with '**future=1**' seems to resolve the issue and generates valid outputs for cron that are truly for future observations.

Cheers

**Comment by darksidelemm** [comment]
Yep this repo is completely abandoned unfortunately. I'm happy to accept a pull request that fixes it, but dont really have time to make and test the fixes myself.

**Comment by satcolintel5** [comment]
Totally understand.  I'll looking into this further. Thank you again
